harbor-client/src/metadata.rs

```diff
@@ -0,0 +1,116 @@
+use crate::http::make_get_request;
+use bitcoin::secp256k1::PublicKey;
+use fedimint_client::ClientHandleArc;
+use fedimint_core::config::FederationId;
+use fedimint_core::module::serde_json;
+use log::error;
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use tokio::sync::RwLock;
+
+/// Global cache of federation metadata
+pub(crate) static CACHE: Lazy<RwLock<HashMap<FederationId, FederationMeta>>> =
```

Comment:
I think this can be a LazyLock? https://doc.rust-lang.org/stable/std/sync/struct.LazyLock.html so we can drop the once_cell dep

Reply:
Can't use this because we need to use the tokio rwlock in threads